rust/noosphere-gateway/src/authority.rs

```diff
@@ -52,16 +56,17 @@ impl GatewayAuthority {
 }
 
 #[async_trait]
-impl<S> FromRequestParts<S> for GatewayAuthority
+impl<S, St> FromRequestParts<St> for GatewayAuthority<S>
 where
-    S: Send + Sync,
+    St: Send + Sync,
+    S: Storage + 'static,
```

Comment:
Just noting that there is no t (you named it `St`) in `Send + Sync` and there are three in `Storage + 'static`, which will probably confuse people w/ naming.

Reply:
Nice catch! The `St` type reflects the underlying axum `State` as rationale for naming, but you make a great point, will update to something less ambiguous

Reply:
I personally wouldn't balk at a non-abbreviated identifier for the generic type.